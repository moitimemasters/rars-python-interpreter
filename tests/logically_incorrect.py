if 1 < 2:
    a += 4
else:
    a += 5
