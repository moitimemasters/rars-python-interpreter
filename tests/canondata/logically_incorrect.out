file: tests/logically_incorrect.py
started lexing
ended lexing
started parsing ast
address of if condition: 268763992
ended parsing ast
started compiling
ended compiling
compiled instructions:
	 Instruction: 2
	 Instruction: 2
	 Instruction: 7
	 Instruction: 6
	 Instruction: 0
	 Instruction: 2
	 Instruction: 7
	 Instruction: 8
	 Instruction: 5
	 Instruction: 0
	 Instruction: 2
	 Instruction: 7
	 Instruction: 8
started interpreting
ERROR: Variable not found
